release-linux-llvm/scripts/build-tiflash-release.sh

```
@@ -60,55 +60,13 @@ SRCPATH=$(
 NPROC=${NPROC:-$(nproc || grep -c ^processor /proc/cpuinfo)}
 ENABLE_THINLTO=${ENABLE_THINLTO:-ON}
 ENABLE_PCH=${ENABLE_PCH:-ON}
-
-INSTALL_DIR="${SRCPATH}/release-linux-llvm/tiflash"
-rm -rf ${INSTALL_DIR} && mkdir -p ${INSTALL_DIR}
-
 if [ $CMAKE_BUILD_TYPE == "RELWITHDEBINFO" ]; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)